actions/pack_action.go

```diff
@@ -30,10 +30,32 @@ type PackAction struct {
 	File             string
 }
 
+
+func (pf *PackAction) Verify(context *debos.DebosContext) error {
+
+	if len(pf.File) == 0 {
+		return fmt.Errorf("Please add a 'file' property.")
+	}
+
+	if len(pf.Compression) > 0 {
+		if pf.Compression != "gz" && pf.Compression != "xz" &&  pf.Compression != "bzip2" {
```

Comment:
here you need to use a map for compression type as well and walk instead of just comparing with names.
Please put the map somewhere outside the functions, so it could be used in both places.

Reply:
Please leave the gz as default option